src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -220,72 +109,49 @@ function WorkspaceInvitePage(props) {
-                        onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)}
-                    >
-                        <HeaderWithBackButton
-                            title={translate('workspace.invite.invitePeople')}
-                            subtitle={policyName}
-                            shouldShowGetAssistanceButton
-                            guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_MEMBERS}
-                            onBackButtonPress={() => {
-                                Policy.clearErrors(props.route.params.policyID);
-                                Navigation.goBack(ROUTES.WORKSPACE_MEMBERS.getRoute(props.route.params.policyID));
-                            }}
-                        />
-                        <SelectionList
-                            canSelectMultiple
-                            sections={sections}
-                            textInputLabel={translate('optionsSelector.nameEmailOrPhoneNumber')}
-                            textInputValue={searchTerm}
-                            onChangeText={(value) => {
-                                SearchInputManager.searchInput = value;
-                                setSearchTerm(value);
-                            }}
-                            headerMessage={headerMessage}
-                            onSelectRow={toggleOption}
-                            onConfirm={inviteUser}
-                            showScrollIndicator
-                            showLoadingPlaceholder={!didScreenTransitionEnd || !OptionsListUtils.isPersonalDetailsReady(props.personalDetails)}
-                            shouldPreventDefaultFocusOnSelectRow={!Browser.isMobile()}
+            {({didScreenTransitionEnd}) => (
+                <FullPageNotFoundView
+                    shouldShow={(_.isEmpty(props.policy) && !props.isLoadingReportData) || !PolicyUtils.isPolicyAdmin(props.policy) || PolicyUtils.isPendingDeletePolicy(props.policy)}
+                    subtitleKey={_.isEmpty(props.policy) ? undefined : 'workspace.common.notAuthorized'}
+                    onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)}
+                >
+                    <HeaderWithBackButton
+                        title={translate('workspace.invite.invitePeople')}
+                        subtitle={policyName}
+                        shouldShowGetAssistanceButton
+                        guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_MEMBERS}
+                        onBackButtonPress={() => {
+                            Policy.clearErrors(props.route.params.policyID);
+                            Navigation.goBack(ROUTES.WORKSPACE_MEMBERS.getRoute(props.route.params.policyID));
+                        }}
+                    />
+                    <MemberInviteList
+                        didScreenTransitionEnd={didScreenTransitionEnd}
```

Comment:
> @barros001 We should not remove didScreenTransitionEnd, the loading components are still rendered but I don't know why they're invisible (You can inspect the page and see the loading components are still there)

Agreed, we need the `didScreenTransitionEnd` (I believe it has todo with focus), but IMO only the page itself should really know about it and not the `MemberInviteList` component. I'm doing some testing here and will update with my findings.

Reply:
@tienifr https://github.com/tienifr/App/pull/35

My understanding is that `didScreenTransitionEnd` is used to delay rendering the member list until the screen fully transition and that was done because there's an input that's auto focused once we render it. Delaying prevents some issue with virtual keyboard (this is just a guess, I did not confirm). I was also expecting some sort of loading indicator during this waiting period, but it really does not show anything. The PR above will simply not render the list altogether (which now also wont render the invite/next button) until we're ready.

The behavior is similar (except that now it also won't render the button) but I'm not sure this is 100% correct. My main thing here is passing a prop called `didScreenTransitionEnd` into a component does not sound correct as a standalone component shouldn't be aware of things like this. Another option is to simply rename it to something else so that it's more semantically correct (something like `shouldLoad` or similar). 

Thoughts?

Btw. there's a bug with `didScreenTransitionEnd` (not related to this PR, it's on staging and production as well - I'll report on slack) and you can see the behavior when its waiting for the transition to end: go to staging, workspace, member, invite user. When on the invite page, refresh the page. It will take you right back to that page but the page will be blank with the disabled invite/next button at the bottom. That's what shows when the page is waiting for the transition to happen)